database/models.py

```diff
@@ -25,6 +25,7 @@ class Experiment(Base):
 
     name = Column(String, nullable=False, primary_key=True)
     time_created = Column(DateTime(), server_default=sqlalchemy.func.now())
+    git_hash = Column(String, nullable=True)
```

Comment:
Do we want to make this nullable?
(If we make it nullable we don't need to deal with migrating existing data and we can handle non-master branches by not including them).
Do we need the branch? maybe get_git_commit should return None if not on master? Maybe it should except if there are uncommitted changes?

Reply:
Yes nullable allows for empty columns, in case we need it for experiment without hashes.
only master might not be doable, since we might want to run experiment on an older commit added assert on uncommitted changes.